airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -6690,29 +6690,66 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-paypal-transaction:0.1.7"
+- dockerImage: "airbyte/source-paypal-transaction:0.1.7.1"
```

Comment:
0.1.8  ?

Reply:
Yes, it will be updated after the publish.